Calls: update FAQ with more information and IP addresses (#15867)
cloudflare/cloudflare-docs: content/calls/turn/faq.md

Cloudflare Calls TURN server runs on [Cloudflare's global network](https://www.cloudflare.com/network) - a growing global network of thousands of machines distributed across hundreds of locations, with the notable exception of the Cloudflare's [China Network](/china-network/).

### Does Cloudflare Calls TURN use the Cloudflare Backbone or is there any "magic" Cloudflare do to speed connection up?

> Cloudflare Calls TURN allocations are homed in the nearest available Cloudflare datacenter to the TURN client via anycast routing. If both ends of a connection is using Cloudflare Calls TURN, Cloudflare will be able to control the routing and, if possible, route TURN packets through the Cloudflare backbone.

Reviewer: ```suggestion
Cloudflare Calls TURN allocations are homed in the nearest available Cloudflare datacenter to the TURN client via anycast routing. If both ends of a connectionare using Cloudflare Calls TURN, Cloudflare will be able to control the routing and, if possible, route TURN packets through the Cloudflare backbone.
```

> Cloudflare Calls TURN allocations are homed in the nearest available Cloudflare datacenter to the TURN client via anycast routing. If both ends of a connection is using Cloudflare Calls TURN, Cloudflare will be able to control the routing and, if possible, route TURN packets through the Cloudflare backbone.

Author: ```suggestion
Cloudflare Calls TURN allocations are homed in the nearest available Cloudflare datacenter to the TURN client via anycast routing. If both ends of a connection are using Cloudflare Calls TURN, Cloudflare will be able to control the routing and, if possible, route TURN packets through the Cloudflare backbone.
```

Cloudflare's [China Network](/china-network/) does not participate in serving Calls traffic and TURN traffic from China will connect to Cloudflare locations outside of China.

### How long does it take for TURN activity to be available in analytics?
TURN usage shows up in analytics in 30 seconds.

## Technical

### I need to allowlist (whitelist) Cloudflare Calls TURN IP addresses. Which IP addresses should I use?
Clouflare Calls TURN is easy to use by IT administrators who have strict firewalls because it requires very few IP addresses to be allowlisted compared to other providers. You must allowlist both IPv6 and IPv4 addresses.

Please allowlist the following IP addresses:
- `2a06:98c1:3200::1/128`
- `141.101.90.1/32`

Cloudflare tries to, but cannot guarantee that the IP addresses used for the TURN service won't change unless this is in your enterprise contract. For more details about static IPs, guarantees and other arrangements please discuss with your enterprise account team. Your enterprise account will be able to provide additional addresses to allowlist as future backup to achieve address diversity.

> If you are allowlisting IP addresses, and don't have a enterprise contract with different instructions, you must set up alerting that detects changes the DNS response from `turn.cloudflare.com` (A and AAAA records) and update the hardcoded IP address(es) accordingly within 14 days of the DNS change.

Reviewer: ```suggestion
If you are allowlisting IP addresses and do not have a enterprise contract with different instructions, you must set up alerting that detects changes the DNS response from `turn.cloudflare.com` (A and AAAA records) and update the hardcoded IP addresses accordingly within 14 days of the DNS change.
```
